CHANGELOG.md

```diff
@@ -35,8 +35,11 @@ For more information about this file see also [Keep a Changelog](http://keepacha
   
 - shiny/Data-Ingest 
   - Download data from DataONE to a temporary directory, display the contents of that directory, and then move files from the temporary directory to dbfiles
-  - Upload files from local machines (via drag and drop on some browsers), display files, and then move these files to a directory in dbfiles. 
-
+  - Upload files from local machines (via drag and drop on some browsers), display files, and then move these files to a directory in dbfiles.
+  - Spinner displays when class "shiny-busy" is invoked during the dataONE download process. In this way, users can be sure that the app is live and is processing their request. 
+  - Users can now input the name of the destination directory that they wish to create within dbfiles. 
+  - Updated Travis.yml to include librdf0-dev so that it can download redland, datapack, and dataone. 
```

Comment:
For the record it's usually OK to leave changes to .travis.yml unmentioned in the changelog -- think of them as a necessary part of the user-facing changes described in the lines above, not a separate feature in themselves. No need to change this one, though -- the extra detail doesn't hurt.

Reply:
Got it. Thanks!!